### DISCUSSION TITLE
minio/oss as online storage。replace fs1

### DISCUSSION THREAD

**Discussion by lxb0425** [dcm4che/dcm4chee-arc-light#3963] [Q&A]
![image](https://user-images.githubusercontent.com/50564015/219589424-ee3343e4-7093-495c-87b5-c4ce9703d1a4.png)
But it didn't take effect
![image](https://user-images.githubusercontent.com/50564015/219589189-0a48bcab-f811-4ea7-b577-b9eaa3449b8d.png)

Whether this configuration is correct is please ？

**Comment by vrindanayak** [comment]
@lxb0425 : See
- [S3 Storage](https://github.com/dcm4che/dcm4chee-arc-light/wiki/S3-Storage)
  - [Amazon S3 Storage](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Amazon-S3-Storage)
